frontend/src/components/filter/activityReportFilters.js

```diff
@@ -342,6 +344,23 @@ export const targetPopulationsFilter = {
   ),
 };
 
+export const singleOrMultiRecipientsFilter = {
+  id: 'singleOrMultiRecipients',
+  display: 'Single or multiple recipients',
+  conditions: SINGLE_OR_MULTI_RECIPIENT_CONDITIONS,
+  defaultValues: {
+    is: 'Single recipient',
```

Comment:
```suggestion
    is: 'single-recipient',
```

otherwise:

https://github.com/HHS/Head-Start-TTADP/blob/ab4a5b2792901f42457021bb1b24e5f035af41bf/src/scopes/activityReport/singleOrMultiRecipient.js#L18

`singleOrMulti` here will get the value ["Single recipient"]